tests/api_workflow/test_api_workflow_upload_dataset.py

```diff
@@ -144,7 +144,7 @@ def failing_upload_sample(*args, **kwargs):
 
     def test_upload_dataset_twice_with_overlap(self):
 
-        all_sample_names = [f'sample_{i}.jpg' for i in range(10)]
+        all_sample_names = [f'img_{i}.jpg' for i in range(10)]
```

Comment:
why did you change this?

Reply:
see https://github.com/lightly-ai/lightly/pull/545/files/4f3e7328539683fdd4990d463da0891aaf92cc4d#r750983079